File: src/main/java/net/igneo/icv/entity/helmet/blackHole/BlackHoleRenderer.java
package net.igneo.icv.entity.helmet.blackHole;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class BlackHoleRenderer extends GeoEntityRenderer<BlackHoleEntity> {
    
    public BlackHoleRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new BlackHoleModel());
    }
}

File: src/main/java/net/igneo/icv/entity/helmet/blackHole/BlackHoleRenderer.kt
package net.igneo.icv.entity.helmet.blackHole

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class BlackHoleRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<BlackHoleEntity?>(renderManager, BlackHoleModel())
